native/common/jp_classhints.cpp

```diff
@@ -244,8 +244,13 @@ class JPHintsConversion : public JPConversion
 	virtual JPMatch::Type matches(JPMatch &match, JPClass *cls)
 	{
 		PyJPClassHints *pyhints = (PyJPClassHints*) cls->getHints();
+		// GCOVR_EXCL_START
+		if (pyhints == NULL)
+			PyJPClass_create(*match.frame, cls);
+		pyhints = (PyJPClassHints*) cls->getHints();
```

Comment:
this should go into the pyhints == NULL block, right?

Reply:
Yeah it is not necessary to look it up twice.  Will fix.